### ISSUE TITLE
I am refactor your shader to new 2021 react component but on every render the shade render a different color

### ISSUE THREAD

**Issue by quanndh** [stoffern/gl-react-instagramfilters#24]

**Comment by cardyy** [comment]
can you rephrase your question, we cant understand it

**Comment by quanndh** [comment]
Basically on every re render a different "filter" is placed on my image. I use the shader in the project example. I had move on to different library so I will close this issue.
